Sources/MySQL/Database/MySQLDatabaseConfig.swift

```diff
@@ -20,12 +20,16 @@ public struct MySQLDatabaseConfig {
     /// Database name.
     public let database: String
 
+    /// Character set. Default utf8_general_ci
+    let characterSet: MySQLCharacterSet
+
     /// Creates a new `MySQLDatabaseConfig`.
-    public init(hostname: String = "127.0.0.1", port: Int = 3306, username: String, password: String? = nil, database: String) {
+    public init(hostname: String = "127.0.0.1", port: Int = 3306, username: String, password: String? = nil, database: String, characterSet: String = "utf8_general_ci") {
```

Comment:
why not accept `MySQLCharacterSet` here if it's public?

Reply:
I think is more convinient use String if you get the values from the Environment.

```swift
let config = try MySQLDatabaseConfig(hostname: Environment.get("HOSTNAME")!, port: Environment.get("PORT")!, username: Environment.get("USERNAME")!, password: Environment.get("PASSWORD")!, database: Environment.get("DATABASE")!, characterSet: Environment.get("CHARACTERSET")!)
```